sources/packages/backend/apps/api/src/route-controllers/overaward/models/overaward.dto.ts

```diff
@@ -17,7 +17,7 @@ import {
 } from "class-validator";
 
 export class OverawardBalanceAPIOutDTO {
-  overawardBalanceValues: Record<string, number>;
+  overawardBalanceValues: Record<string, number | undefined>;
```

Comment:
When the key is present, the value cannot be undefined. The existing type is accurate.

Reply:
Reverted.